src/Admin/Admin.php

```diff
@@ -152,13 +155,19 @@ protected function get_assets(): array {
 			return ( null !== $screen && 'product' === $screen->id );
 		};
 
-		$assets[] = ( new AdminScriptAsset(
-			'gla-custom-inputs',
-			'js/build/custom-inputs',
-			[],
-			'',
+		$assets[] = ( new AdminScriptWithBuiltDependenciesAsset(
+			'gla-product-attributes',
+			'js/build/product-attributes',
+			"{$this->get_root_dir()}/js/build/product-attributes.asset.php",
+			new BuiltScriptDependencyArray(
+				[
+					'dependencies' => [],
+					'version'      => (string) filemtime( "{$this->get_root_dir()}/js/build/product-attributes.js" ),
+				]
+			),
 			$product_condition
-		) );
+		) )->add_inline_script( ...$gla_data_inline_script_args );
```

Comment:
Thanks for the suggestion.

It's uncertain if the new Product Edit page will have the case of setting `glaData` twice. Indeed, it will be better to avoid possible conflicts in advance. Separated the inline script withe different variable names in https://github.com/woocommerce/google-listings-and-ads/pull/2087/commits/0468a73eff2460467a569d0f3c29b385499b1560.